app/controllers/orders_controller.rb

```diff
@@ -51,14 +54,19 @@ def destroy
 		end
 	end 
 
+	def sender_orders
+	    @orders = Order.where(:user_id => current_user.id)
```

Comment:
use this syntax
`Order.where(user_id: current_user.id)`

Reply:
done